src/redirects.json

```diff
@@ -675,6 +675,10 @@
         "link": "/policy/privacy",
         "redirect": "/privacy"
     },
+    {
+        "link": "/cli/install.sh",
+        "redirect": "https://appwrite.io/cli/install.ps1"
```

Comment:
Did this work for you on windows?

Reply:
It did not, but after changing it to the following: 
```
{
   "link": "/cli/install.ps1",
   "redirect": "https://raw.githubusercontent.com/appwrite/sdk-for-cli/master/install.ps1"
}
```
I recieve the following error message on testing the windows powershell script cli install command:
```
iwr -useb http://localhost:5173/cli/install.ps1 | iex
iwr : The remote server returned an error: (308) Permanent Redirect.
At line:1 char:1
+ iwr -useb http://localhost:5173/cli/install.ps1 | iex
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```
So I tried to make sure `curl -sL https://appwrite.io/cli/install.sh | bash` was working properly. But I also recieve the following error message:
```
$ curl -sL https://appwrite.io/cli/install.sh | bash
     _                            _ _           ___   __   _____
    /_\  _ __  _ ____      ___ __(_) |_ ___    / __\ / /   \_   \
   //_\\| '_ \| '_ \ \ /\ / / '__| | __/ _ \  / /   / /     / /\/
  /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_
  \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/
        |_|   |_|


🔥 Welcome to the Appwrite CLI install shield 🛡
[1/4] Getting System Info ...

OS : mingw64_nt-10.0-22631
ARCH : x64
REQUIRES ROOT : false

[2/4] Downloading executable for mingw64_nt-10.0-22631 (x64) ...
🚦 Downloading Appwrite CLI 5.0.5 ...
❌ Couldn't find executable for mingw64_nt-10.0-22631 (x64). Please contact the Appwrite team
```

Reply:
The link shouldn't be install.sh.